hack/ginkgo-e2e.sh

```diff
@@ -102,6 +102,7 @@ export PATH=$(dirname "${e2e_test}"):"${PATH}"
   --repo-root="${KUBE_ROOT}" \
   --node-instance-group="${NODE_INSTANCE_GROUP:-}" \
   --prefix="${KUBE_GCE_INSTANCE_PREFIX:-e2e}" \
+  --os-distro="${OS_DISTRIBUTION:-debian}" \
```

Comment:
I'd prefer keeping defaults out of this file, leaving them only in e2e.go. instead do something like
```shell
${OS_DISTRIBUTION:+"--os-distro=${OS_DISTRIBUTION}"} \
```
so that we only include this flag when `$OS_DISTRIBUTION` is set.